content/applications/sales/sales/ebay_connector/manage.rst

```diff
@@ -2,49 +2,115 @@
+users pay for items on eBay through Odoo. Therefore, once orders are synced into Odoo from eBay they
+are already paid for, and Odoo's invoice / payment functionality will not need to be utilized.
+However, invoices need to be created and marked as Paid to “close” the sale order.
+
+Users can opt to mass create and post invoices in batches. To do so, navigate to Quotations in the
+list view by going to :menuselection:`Sales app --> Orders --> Quotations`. In the upper right
+corner select the list view icon. Then check the boxes that invoices should be made for and go to
+the :guilabel:`Action` menu. Click on :guilabel:`Create Invoices`.
+
+A pop-up will appear and click on :guilabel:`Create and view invoice`. A new screen will populate
+with the newly created invoices. Next, select all of them by clicking on the box icon next to
+:guilabel:`Number` in the header row of the list, this will select all the records. Then navigate to
+the :guilabel:`Action` menu and click :guilabel:`Post entries`. Following this step, a pop-up will
+appear and click on :guilabel:`Post journal entries`. This will take the invoices out of *draft* and
+set them to *posted*.
+
+Reconciling payments
+--------------------
+
+Users typically utilize PayPal to receive payment from eBay, and then send lump sums from PayPal
+into their bank account. To reconcile this income, users can reconcile the one PayPal transfer
+with all related invoices.
+
+First navigate to the :guilabel:`Accounting Dashboard` by going to the :menuselection:`Accounting
+app --> Dashboard --> Bank`. :guilabel:`Create` a new transaction and enter the :guilabel:`Label`
+as `eBay Sales`. Fill out the :guilabel:`Amount` and enter a :guilabel:`Statement` date in. Click on
+:guilabel:`Create and edit`.
+
+For the :guilabel:`Ending Balance` enter the same account that was entered for the
+:guilabel:`Amount` above. Click on :guilabel:`Save`. Next, open the new balance that needs to be
+reconciled. Under the tab marked: :guilabel:`Match Existing Entries` select the entries that are
+included in this balance.
 
-Products identifiers such as EAN, UPC, Brand or MPN are required in most of the eBay category.
-The module manages the EAN and UPC identifiers with the **Barcode** field of the product variant.
-If the **Barcode** field is empty or is value is not valid, the EAN and UPC values will be set as 'Does not apply' as recommended by eBay.
-The Brand and MPN values are working as item specifics and should be define in the **Variants** tab on the product form.
-If these values are not set, 'Does not apply' will be used for the eBay listing.
+Finally, click :guilabel:`Validate` after all of the appropriate entires are added in. The
+reconciliation is complete, now navigate to the customer invoices by going to
+:menuselection:`Customers --> Invoices`. The **Paid** label should appear under the
+:guilabel:`Payment Status` column.
```

Comment:
Slight rewording and switching clauses around. Totally optional, Tim :smile: 

```suggestion
After adding all the necessary entries, click :guilabel:`Validate` to complete the reconciliation.
To verify the payment, navigate to :menuselection:`Customers --> Invoices` and select the desired
customer invoice. The **Paid** label should appear under the :guilabel:`Payment Status` column.
```

I feel like this paragraph shows the difference in our writing styles, so I just wanted to share how I would've described what's happening. Feel free to go with what you had originally, since there was nothing wrong! :blush: 

When I read your writing, I see step-by-step how-to guide --> Do x, then after that's done, do y.
For me, I tend to explain why you even want to do x in the first place, and then doing x makes y happen.

If that makes sense. Sorry if I seem like I'm imposing my writing style on you! I just wanted to share how I'd describe something, and what I'm looking for when reviewing. And I'm learning a lot from reviewing your docs! :grin: